### PR TITLE
Task 12 localization

### PR DESCRIPTION
**Pull request by universezzz**

### CODE REVIEW

File: src/containers/profile/account/index.jsx [hunk 1]
```diff
@@ -29,7 +30,7 @@ class Account extends Component {
         password: '',
         confirmPassword: '',
         errors: {},
-        update: !this.state.update
+        update: !this.state.update,
```
Comment (melonholy) [inline src/containers/profile/account/index.jsx]: i recommend you to use functional set state with previous state param

**Review by melonholy — CHANGES_REQUESTED** [review]

File: src/components/courseCard/index.js [hunk 2]
```diff
@@ -3,15 +3,16 @@ import PropTypes from 'prop-types';
...
 import { links } from '../../utils/constants';
 
 import styles from './style.module.scss';
 
-export const CourseCard = course => {
-  const { title, authors, rate, price = 'For free', img = '', id } = course;
-
+const CourseCard = props => {
+  const { title, authors, rate, price = 'For free', img = '', id, t } = props;
+
```
Comment (OlegPanasyuk) [inline src/components/courseCard/index.js:15]: what does it mean 't'? Is it possible to replace this variable with 'localization' or 'translator' naming?

**Review by OlegPanasyuk — APPROVED** [review]

File: src/components/courseCard/index.js (hunk 2 above)
Comment (universezzz) [reply src/components/courseCard/index.js:15]: This is a function from i18 library, the custom name is 't'. Do you wanna rename it to 'translator'?

Comment (OlegPanasyuk) [reply src/components/courseCard/index.js:15]: Yep, of course, if it is possible.) On my mind, It is more readable.

**Review by OlegPanasyuk — APPROVED** [review]

**Review by slavasgod — APPROVED** [review]

**Pull request merged**